### DISCUSSION TITLE
聚一场散一场

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#10] [General]
# 聚一场散一场

不要对任何人，有道德上的洁癖，这个世界上的任何灵魂，都是半人半鬼，凑得太近谁都没法看。

https://www.edony.ink/impermanence/

**Comment by edonyzpc** [comment]
最近状态不好，这一期本来打算就这么鸽掉的，但是想了想，十年之约这才半年啊！
所以就水一下，说说最近自己思考的一个问题吧。

**Comment by hinzer** [comment]
感谢分享，通过阅读您的文字我觉得能拓展自己的见识！

**Comment by hinzer** [comment]
能理清自己的想法和诉求和理解他人我觉得就很优秀了，管理团队要照顾到很多人的需求一定也不容易。

**Reply by edonyzpc** [reply]
确实是的，管理是一门艺术，需要很好的能力。不过这不影响骂老板政治正确这件事情🤪
